### PR TITLE
Split parse and stringify to separate packages

### LINKED ISSUE #181: Simplify parse and stringify methods
Atm naming has shared parts of code with BemEntityName like usage of deprecated fields.
Would be better to has a tiny method for stringifying entityName with naming:
```js
let res = e.block;
e.elem && (res += delims.elem + e.elem);
e.mod && (
  res += delims.mod.name + e.mod.name,
  e.mod.val === true || (res += delims.mod.val + e.mod.val)
);
return res;
```
and another tiny method for normalization old versions to the modern one as it does in BemEntityName.create: https://github.com/bem-sdk/bem-entity-name/blob/86e59d24/lib/entity-name.js#L379-L406

Perhaps we can make a separate function to let users use it with stringify as they did it before.

### PR DESCRIPTION
**Pull request by qfox**
Fixes #181

### CODE REVIEW

File: packages/graph/spec/natural-order/ordering.spec.js [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+
+    const resolved = Array.from(graph.naturalDependenciesOf(decl));
+
+    const indexA = findIndex(resolved, { entity: { block: 'A' } });
+    const indexAelem = findIndex(resolved, { entity: { block: 'A', elem: 'e' } });
+
+    expect(indexAelem).to.be.below(indexA);
+});
+
+/*test('should place ordered entity from decl before several entities depending on it', () => {
```
Comment (tadatuta) [inline packages/graph/spec/natural-order/ordering.spec.js]: ;)

File: packages/naming.entity.stringify/test/stringify.test.js [hunk 2]
```diff
@@ -3,10 +3,10 @@
 const test = require('ava');
 const BemEntityName = require('@bem/entity-name');
 
-const naming = require('../index');
+const stringify = require('../index')(require('@bem/sdk.naming.entity.presets/origin'));
```
Comment (tadatuta) [inline packages/naming.entity.stringify/test/stringify.test.js]: code style is inconsistent with https://github.com/bem/bem-sdk/pull/211/files#diff-caf07bbad4c7fd6c0f9efbdb8016717dR5

File: packages/decl/lib/assign.js [hunk 3]
```diff
@@ -62,5 +56,6 @@ module.exports = function (file, scope) {
         result.entity.modVal = isValidVal(fEntity.modVal) ? fEntity.modVal : sEntity.modVal;
     }
 
-    return cellify(result);
+console.log('aoasdobasd', { file, scope, result }, BemCell.create(result));
```
Comment (qfox) [inline packages/decl/lib/assign.js]: ---

**Comment by qfox** [comment]
Fixes #181

File: packages/bemjson-to-decl/index.js [hunk 4]
```diff
@@ -1,8 +1,8 @@
 'use strict';
 
 const stringifyObj = require('stringify-object');
-const normalize = require('bem-decl').normalize;
-const BemEntity = require('@bem/entity-name');
+const normalize = require('@bem/sdk.decl').normalize;
+const BemEntity = require('@bem/sdk.entity-name');
```
Comment (tadatuta) [inline packages/bemjson-to-decl/index.js:5]: is it still proper name for constant?
btw, it's called `b_` in helpers in the same package and `BemEntityName` bellow (which sounds like the best option to me)

File: packages/bemjson-to-jsx/lib/index.js [hunk 5]
```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-var bn = require('@bem/sdk.naming');
+var bn = require('@bem/sdk.naming.entity');
```
Comment (tadatuta) [inline packages/bemjson-to-jsx/lib/index.js:3]: `bn`, `bemNaming` and `BemNaming` in different packages

**Review by tadatuta — APPROVED** [review]

File: packages/bemjson-to-jsx/lib/index.js (hunk 5 above)
Comment (qfox) [reply packages/bemjson-to-jsx/lib/index.js:3]: → https://github.com/bem/bem-sdk/issues/212

File: packages/bemjson-to-decl/index.js (hunk 4 above)
Comment (qfox) [reply packages/bemjson-to-decl/index.js:5]: It's a constructor so yes.

Comment (qfox) [reply packages/bemjson-to-decl/index.js:5]: `new b_()` ?

**Pull request merged**